app/screens/channel_info/channel_info.js

```diff
@@ -464,7 +464,7 @@ export default class ChannelInfo extends PureComponent {
                         <View style={style.footer}>
                             <ChannelInfoRow
                                 action={this.handleDelete}
-                                defaultMessage='Delete Channel'
+                                defaultMessage='Archive Channel'
                                 icon='trash'
```

Comment:
Just wondering if icon should change as well (like to `archive`)?

Reply:
@esethna what would you like me to do here?

Reply:
Good point, thanks. @grundleborg can we use an archive icon instead of the trash can? We use the archive icon on webapp, we can go with something similar (not sure what's available): 
![image](https://user-images.githubusercontent.com/14333569/41732548-8c42019c-753e-11e8-88a3-bb4491c48102.png)